### PR TITLE
Added Changelog for Feb and March

### PR DESCRIPTION
**Pull request by DannyRuchtie**
Added Changelog 

**February**
Features

+ Anti Phishing banner
+ Keyboard traversing in file tree @jyash97
+ Logging for errors when installing private package
+ New Over-mind typing
+ Custom Cover Images for sandboxes

Fixes

- Export ordering when re-exporting an import
- Yarn lockfile was updated @NullVoxPopuli

**March**
Features

+ Improve esmodule detection
+ Preload babel.min.js from root
+ Github pages deploy
+ Added Shared with me & Liked page
+ Add Basic auth as option in private npm
+ Add Podcast Pages

Fixes
- Vue in Safari
- Updated lint warnings
- Console.log in SSE and static sandboxes
- Prevent back & forward navigation on horizontally swipe

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/5653/commits/9d43614bc4d33e70be2499d562490e5dfc5e671f is at https://pr5653.build.csb.dev/s/new.

File: packages/homepage/content/changelog/4-0-1/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+banner: ./4-0-1.jpg
+title: v4.0.1
+slug: Custom Artwork
+date: 2021-02-23
+description: Add custom thumbnails to your sandboxes
+---
+
+#### Features
+
+1. Anti Phishing banner
```
Comment (CompuIves) [inline packages/homepage/content/changelog/4-0-1/index.md:11]:
```suggestion
1. Add a warning banner to sandboxes that are more likely to be phishing
```

File: packages/homepage/content/changelog/4-0-1/index.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+date: 2021-02-23
+description: Add custom thumbnails to your sandboxes
+---
+
+#### Features
+
+1. Anti Phishing banner 
+2. Keyboard traversing in file tree [@jyash97](https://github.com/codesandbox/codesandbox-client/pull/5505)
+3. Logging for errors when installing private package 
+4. New Over-mind typing
```
Comment (CompuIves) [inline packages/homepage/content/changelog/4-0-1/index.md:14]:
```suggestion
4. Improvements in our state management 
```

File: packages/homepage/content/changelog/4-0-1/index.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+slug: Custom Artwork
+date: 2021-02-23
+description: Add custom thumbnails to your sandboxes
+---
+
+#### Features
+
+1. Anti Phishing banner 
+2. Keyboard traversing in file tree [@jyash97](https://github.com/codesandbox/codesandbox-client/pull/5505)
+3. Logging for errors when installing private package
```
Comment (CompuIves) [inline packages/homepage/content/changelog/4-0-1/index.md:13]:
```suggestion
3. More helpful error messages for errors while installing a private package
```

File: packages/homepage/content/changelog/4-0-1/index.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+1. Anti Phishing banner 
+2. Keyboard traversing in file tree [@jyash97](https://github.com/codesandbox/codesandbox-client/pull/5505)
+3. Logging for errors when installing private package 
+4. New Over-mind typing 
+5. Custom Cover Images for sandboxes 
+
+
+
+#### Fixes
+- Export ordering when re-exporting an import
```
Comment (CompuIves) [inline packages/homepage/content/changelog/4-0-1/index.md:20]:
```suggestion
- Bundler: fix the export ordering when you're re-exporting an import
```

File: packages/homepage/content/changelog/4-0-1/index.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+2. Keyboard traversing in file tree [@jyash97](https://github.com/codesandbox/codesandbox-client/pull/5505)
+3. Logging for errors when installing private package 
+4. New Over-mind typing 
+5. Custom Cover Images for sandboxes 
+
+
+
+#### Fixes
+- Export ordering when re-exporting an import 
+- Yarn lockfile was updated [@NullVoxPopuli](https://github.com/codesandbox/codesandbox-client/pull/5501)
```
Comment (CompuIves) [inline packages/homepage/content/changelog/4-0-1/index.md:21]: This is more of an internal maintenance, so I would not put this one in explicitly.

```suggestion
```

File: packages/homepage/content/changelog/4-0-2/index.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+date: 2021-03-21
+description: Likes and shared with you, now in the dashboard and Publish to GitHub Pages
+---
+
+#### Features
+
+1. Improve esmodule detection 
+2. Preload babel.min.js from root 
+3. Github pages deploy 
+4. Added Shared with me & Liked page
```
Comment (CompuIves) [inline packages/homepage/content/changelog/4-0-2/index.md:14]:
```suggestion
4. Added Shared With Me & Liked page in the dashboard. Now you can see which sandboxes have been shared with you and which ones you've liked before. 
```

File: packages/homepage/content/changelog/4-0-2/index.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+description: Likes and shared with you, now in the dashboard and Publish to GitHub Pages
+---
+
+#### Features
+
+1. Improve esmodule detection 
+2. Preload babel.min.js from root 
+3. Github pages deploy 
+4. Added Shared with me & Liked page 
+5. Add Basic auth as option in private npm
```
Comment (CompuIves) [inline packages/homepage/content/changelog/4-0-2/index.md:15]:
```suggestion
5. Add Basic auth support in private npm, making it possible to use private npm with Azure DevOps and Artifactory
```

File: packages/homepage/content/changelog/4-0-2/index.md [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+title: v4.0.2
+slug: Custom Artwork
+date: 2021-03-21
+description: Likes and shared with you, now in the dashboard and Publish to GitHub Pages
+---
+
+#### Features
+
+1. Improve esmodule detection 
+2. Preload babel.min.js from root
```
Comment (CompuIves) [inline packages/homepage/content/changelog/4-0-2/index.md:12]:
```suggestion
2. Improve initial loading speed of the bundler
```

File: packages/homepage/content/changelog/4-0-2/index.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+banner: ./4-0-2.jpg
+title: v4.0.2
+slug: Custom Artwork
+date: 2021-03-21
+description: Likes and shared with you, now in the dashboard and Publish to GitHub Pages
+---
+
+#### Features
+
+1. Improve esmodule detection
```
Comment (CompuIves) [inline packages/homepage/content/changelog/4-0-2/index.md:11]:
```suggestion
1. Improve esmodule detection in the bundler
```

File: packages/homepage/content/changelog/4-0-2/index.md [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+slug: Custom Artwork
+date: 2021-03-21
+description: Likes and shared with you, now in the dashboard and Publish to GitHub Pages
+---
+
+#### Features
+
+1. Improve esmodule detection 
+2. Preload babel.min.js from root 
+3. Github pages deploy
```
Comment (CompuIves) [inline packages/homepage/content/changelog/4-0-2/index.md:13]:
```suggestion
3. Added support for deploying to GitHub Pages
```

File: packages/homepage/content/changelog/4-0-2/index.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
+---
+banner: ./4-0-2.jpg
+title: v4.0.2
+slug: Custom Artwork
+date: 2021-03-21
+description: Likes and shared with you, now in the dashboard and Publish to GitHub Pages
```
Comment (CompuIves) [inline packages/homepage/content/changelog/4-0-2/index.md:6]:
```suggestion
description: Likes and Shared With You pages in the dashboard and the ability to publish to GitHub Pages
```

**Review by CompuIves — COMMENTED** [review]
It's super great that you're picking this up, we were really getting behind! I reworded some of the points to add more context for someone reading the changelog.

**Pull request merged**